packages/itinerary-body/src/styled.tsx

```diff
@@ -483,6 +467,10 @@ export const PlaceSubheader = styled.div`
   font-weight: 300;
   padding-left: 4px;
   padding-top: 1px;
+
+  /* Reduce vertical space and fix horizontal alignment of stop id and stop viewer link for transit stops. */
+  margin-top: -14px;
+  margin-left: -4px;
```

Comment:
Instead of the negative `margin-left` here, can you just take off the `padding-left` above?

Reply:
That's a really good catch thank you!